### ISSUE TITLE
v3

### ISSUE THREAD

**Issue by lorenbrichter** [lorenbrichter/ragmask#8]
Placeholder issue, will post experiments with all raw edges eliminated, loops instead of straps, possible filter inserts, etc.

**Comment by lorenbrichter** [comment]
Current v3 based on pro tip 3: https://ragmask.com/pro/ (eliminated last ragged edge). Could modify to incorporate loops (made of ribbon?) instead of straps. Place loops in as part of first fold, clean look.

![IMG_2715](https://user-images.githubusercontent.com/40109/78588407-93a18c00-780c-11ea-85a1-2a708e8b2841.jpeg)

![IMG_2716](https://user-images.githubusercontent.com/40109/78588413-9603e600-780c-11ea-9479-f54d7be12a1f.jpeg)

![IMG_2717](https://user-images.githubusercontent.com/40109/78588419-99976d00-780c-11ea-8a84-23d105b7e121.jpeg)

**Comment by lorenbrichter** [comment]
Leaving a gap in the first stitch makes it possible is insert/remove nose wire at any time

**Comment by lorenbrichter** [comment]
No seam version, single cut, single seam

![IMG_8002](https://user-images.githubusercontent.com/40109/78588760-2b9f7580-780d-11ea-8374-fc9b787cef78.JPG)

**Comment by drf5n** [comment]
Please add a tickmark and a callout on the pattern's ruler for the length of the nosewire.

**Comment by lorenbrichter** [comment]
Awesome idea, can pull this into 2.x

**Comment by Its-Gardner** [comment]
Using a continuous strap long enough for bottom straps and bottom binding eliminates the raw bottom edge, improves the seal and comfort, and hides the opening for the nose wire. I will alter assembly/sewing process to make production as efficient as possible.

![IMG_8184](https://user-images.githubusercontent.com/8468667/79020590-3af12c80-7b47-11ea-97c5-cf4532629105.JPG)
![IMG_8185](https://user-images.githubusercontent.com/8468667/79020597-40e70d80-7b47-11ea-8dfb-b8ca22da9d08.JPG)
![IMG_8186](https://user-images.githubusercontent.com/8468667/79020604-45abc180-7b47-11ea-953a-df06ab3a3d06.JPG)

**Comment by Its-Gardner** [comment]
Looped version with separate ties and bound bottom edge. Conserves microfiber material and may hold up better in washer/dryer?
![55D51220-7F16-4508-80D4-2631C49F163B](https://user-images.githubusercontent.com/8468667/79053634-d9939100-7c0c-11ea-814f-6382dcf5e7cb.JPG)
![IMG_8223](https://user-images.githubusercontent.com/8468667/79053640-def0db80-7c0c-11ea-8e4b-385f09d86e73.JPG)

**Comment by tristanbo** [comment]
A suggestion regarding the straps :  I use 2 looped straps instead of 4 separate straps 
- smooth usability
- you only need to knot once at the back of the head
- no change in sewing of straps

![PastedGraphic-1](https://user-images.githubusercontent.com/63233411/79669359-484b8e00-81bb-11ea-8526-ff756dbef36b.png)
![PastedGraphic-4](https://user-images.githubusercontent.com/63233411/79669368-4eda0580-81bb-11ea-9645-eb7af8308760.png)

**Comment by tristanbo** [comment]
Another suggestion: I place the wire at the last moment, by making a very small opening straight in the fabric (at a small distance from an ending. Removing and replacing it is is easy.
![IMG_9449](https://user-images.githubusercontent.com/63233411/79669577-c65c6480-81bc-11ea-870f-915cf19092c7.JPG)
![IMG_9450](https://user-images.githubusercontent.com/63233411/79669584-cd837280-81bc-11ea-9d5c-2adcb7f7139e.JPG)

**Comment by tristanbo** [comment]
Another suggestion regarding breathability

you all know the situation: 
• when the mask has a lot of "loose space at the front, it's more breathable at first, but soon after it collapses on the face (symmetrically or not) when inspiring, and in the end it's not that much more breathable anymore. Feels like losing 20% breathing capacity compared to no mask.
• when the mask is very much adjusted to the face, there is less leeway, and you breathe with your mouth directly on the fabric. It feels like losing 30-40% breathing capacity compared to no mask.  

By using the current template (and some leftover masks) I did some tests to _increase_ and _take control_ of the  the _structure_ of the fabric when breathing. Early experiments are quite promising: it's VERY breathable, and stays so. 
Basically, the technique is to add 2-3 small sewing lines perpendicular to the central 'vertical sew' of the mask. It creates a 3-dimensionally structured surface with air channels, which doesn't stick to the mouse/nose, and feels like only losing 10-15% breathing compared to no mask.  

![IMG_9575](https://user-images.githubusercontent.com/63233411/79671093-f3fadb00-81c7-11ea-9dd3-d314bebbf4a1.JPG)

![IMG_9576](https://user-images.githubusercontent.com/63233411/79671095-fbba7f80-81c7-11ea-82d9-823ae7fcc032.JPG)

![image](https://user-images.githubusercontent.com/63233411/79671106-10971300-81c8-11ea-9195-c655174bd5f6.png)
